Modules/GLO/glo-itstpc-mtch-qcmn-test.json

```diff
@@ -33,35 +32,33 @@
         "className": "o2::quality_control_modules::glo::ITSTPCMatchingTask",
         "moduleName": "QcGLO",
         "detectorName": "GLO",
-        "cycleDurationSeconds": "60",
+        "cycleDurationSeconds": "1",
```

Comment:
FYI, the framework will cap this at `10` anyway. In any case, please do not use less than `60` in production.`